content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,82 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports will be captured by default.
```

Comment:
Link to https://github.com/istio/istio/pull/16414 or https://preliminary.istio.io/faq/traffic-management/#controlling-inbound-ports?

Reply:
thanks frank will link to the latter.

Reply:
accepted.